.babelrc

```diff
@@ -1,12 +1 @@
-{
-  "presets": ["@babel/env", "@babel/react"],
-  "plugins": [
-    "@babel/plugin-syntax-nullish-coalescing-operator",
-    ["module-resolver", {
-      "root": ["./src"],
-      "alias": {
-        "topo": "./src"
-      }
-    }]
-  ]
-}
+../.babelrc
```

Comment:
This breaks support for builds

Reply:
Should be fixed now, I copied it straight from the masters branch .babelrc files.